docs/source/developing-volttron/developing-agents/using-asyncio-in-agents.rst

```diff
@@ -41,9 +40,9 @@ create tasks from the aforementioned coroutine and run the Event Loop. An exampl
 
 
 Next, the method, 'self._configure_ven_client', contains code to configure our agent including running the Event Loop defined above.
-When Volttron starts the agent, it will call this method. This method is where we use Gevent with Asyncio. Gevent will spawn a coroutine
-(i.e. `greenlet <https://greenlet.readthedocs.io/en/latest/greenlet.html#what-is-greenlet>`_); the Event Loop from Asyncio will be run
-inside that coroutine. Below is the relevant snippet of the callback function:
+When Volttron starts the agent, it will call this method. This method is where we use Gevent with Asyncio. Gevent will spawn a `greenlet
+<https://greenlet.readthedocs.io/en/latest/greenlet.html#what-is-greenlet>`_
```

Comment:
Do you define event loop anywhere?  I know what it is but does the user?

Perhaps it would be good to draw or use an image to what an event loop is and the fact that the async event loop is inside a greenlet
![image](https://user-images.githubusercontent.com/3979063/153655730-eb907930-b2a0-4cc8-aa7b-51218d83a666.png)

Reply:
I like just the text :)